docs/docfiles/user_guides/manual/dpctl/queues.rst

```diff
@@ -0,0 +1,19 @@
+.. _queues:
+
+#####
+Queue
+#####
+
+A queue is used to specify a device and a specific set of features of that
```

Comment:
I think we should take an entry from basic concepts section verbatim to example what a queue is:

> A queue is needed to schedule execution of any computation, or data copying on the device. Queue construction requires specifying a device and a context targeting that device as well as additional properties, such as whether profiling information should be collected or whether submitted tasks are executed in the order in which they were submitted.